src/Pages/Saksoversikt/Saksfilter/Virksomhetsmeny/Virksomhetsmeny.tsx

```diff
@@ -50,11 +52,14 @@ export const Virksomhetsmeny = ({
     );
 
     const [søketreff, setSøketreff] = useState<undefined | Set<string>>(undefined);
+    const [søkeordState, setSøkeordState] = useState<string>('');
```

Comment:
lurer på om `søkeordState === ''` dekkes av `søketreff === undefined` og at vi ikke trenger en ny state for søkeord 🤔

Reply:
Gikk litt fort da jeg så på setSøketreff lengre nede, men ser at den alltid er et sett så da bør det gå bra ja. 
Tester :)